frontend/public/code.css

```diff
@@ -0,0 +1,87 @@
+pre code.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 1em;
+}
+
+code.hljs {
+  padding: 3px 5px;
+}
+
+.hljs {
+  color: #e0e2e4;
+}
+
+.hljs-keyword,
```

Comment:
We could probably stand to break a few of these up, and add some more colors. It's worth doing some combination of:

- [ ] Opening an issue to document that this should be done – we shouldn't block this PR for it
- [ ] Generating some themes ([possibly with HueMint](https://huemint.com/illustration-3/))
- [ ] Syncing with Neil about this if he has time to put together a palette

Reply:
Created issue #267